MLH-Fellowship/VehiCO
# Offset value display on modal and dynamic rendering of routes

## Description
**Current Behavior:**

- No offset information displayed.
- Routes are not rendered dynamically as the travel mode is changed on tabbar.

**Proposed Changes:**

- Displaying routes dynamically on switching travel modes from tabbar.
- Clearing all previous routes from map.
- Displaying modal on clicking on `offset` button on the Sidebar.
- Displaying offset value as  `cf*12` and redirecting to donation site on clicking on donate.

### Issue Number(s): 
Closes #14.

## Screenshots (if appropriate):
![image](https://user-images.githubusercontent.com/42820001/101295469-e45a0080-3843-11eb-9adc-ce09b16afc21.png)


## How Has This Been Tested?


- Check by switching travel modes on the tabbar.
- Check by clicking on Offset button on the Sidebar.

## Pull request type
 
Please check the type of change your PR introduces:

## Checklist
Please check if your PR fulfills the following requirements:
- [X] Docs have been reviewed and added / updated if needed
- [X] My code follows the code style of this project and lint has passed locally
